### PR TITLE
r-saige - rebuild with patched dependency

### PR DESCRIPTION
**Pull request by matuskosut**
rebuild with patched dependency -- bgenix got rebuilt to include missing patch

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by matuskosut** [comment]
@BiocondaBot please add label

File: recipes/r-saige/meta.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ source:
     - patches/0001-use-shared-libs-in-makevars.patch
 
 build:
-  number: 0
+  number: 3
```
Comment (bgruening) [inline recipes/r-saige/meta.yaml:18]: please use 1 instead

Comment (matuskosut) [reply recipes/r-saige/meta.yaml:18]: @bgruening sorry, there is a reason why I ended up with 3.

I tried with 1 first but CI complained that build 1 already exists for this version: https://github.com/bioconda/bioconda-recipes/pull/32545/commits/cae74ff031fd60cbb5bd0c9606240f055066f7de

Any idea why there are builds: `1` and `2` in the conda repo, that were not committed in recipe? 
https://anaconda.org/bioconda/r-saige/files

Comment (bgruening) [reply recipes/r-saige/meta.yaml:18]: Oh! Yes, please look at the bulk branch. @dpryan79 is currently rebuilding packages there. But this also means build 1 or 2 are recent rebuilds and this PR is not needed anymore, left?

Comment (matuskosut) [reply recipes/r-saige/meta.yaml:18]: @bgruening I actually need it, unfortunately I was patching the dependency after the last rebuild so it is not included,

Comment (bgruening) [reply recipes/r-saige/meta.yaml:18]: I see, please do not patch your code, but release a new version next time.

**Review by bgruening — APPROVED** [review]

**Pull request merged**